_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw.cpp

```diff
@@ -198,6 +196,8 @@ mfxStatus MFXVideoENCODEVP9_HW::Init(mfxVideoParam *par)
 
     m_video = *par;
 
+    m_pCore->GetHWType();
```

Comment:
same

Reply:
This call is done this way on purpose. During this call MSDK Core is being initialized that is the mandatory step to keep working. Feel free to suggest an alternative!